reference/v8.json

```diff
@@ -158,17 +158,17 @@
     "maxzoom": {
       "type": "number",
       "default": 14,
-      "doc": "Maximum zoom to preserve detail at."
+      "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels but slower performance)."
```

Comment:
I myself have a pretty vague understanding of the implications of increasing the maxzoom of a GeoJSON tile source. Maybe you could take a swing at writing this doc property?